src/app.less

```diff
@@ -49,11 +49,32 @@
 	}
 
 	.break-all {
-		word-break: break-all;
+		width: 280px;
+		word-break: normal;
+		hyphens: auto;
 	}
 
 	@media (max-width: 1232px) {
 		padding: 0 1rem;
 	}
 
+	.kilde-tekst {
+		color: #645f5a;
+		margin-top: 3px;
+		line-height: 1rem;
+		hyphens: auto;
+		word-break: normal;
+	}
+
+	.personalia-grid {
+		display: grid;
+		gap: 1rem;
+		grid-auto-flow: column;
+	}
+
+	.ekspanderbartPanel {
+		&__innhold {
+			padding: 0 1rem;
```

Comment:
@sneha-d-desai Den her ser ut til å ødelegge litt for paddingen i panelene når det ikke er noe innhold der, de blir limt til bunnen: 
![image](https://user-images.githubusercontent.com/85442404/134019757-f14f14fe-fd39-4f62-861f-91de4524806a.png)
Går det an å løse det du ville med en enda mer spesifikk regel?

Reply:
Jeg fikser det